generated/thrift/rpc.thrift

```diff
@@ -92,6 +93,22 @@ struct WriteRequest {
 	3: required Datapoint datapoint
 }
 
+struct FetchMetadataBatchRawRequest {
+	1: required binary nameSpace
+	2: required list<binary> ids
```

Comment:
could you rename to what type of id this is, i.e. `seriesIDs` or `shardIDs`

Reply:
Yeah `id` is our standard name for series IDs when it comes to the RPC layer.